config/environments/production.rb

```diff
@@ -83,3 +83,6 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 end
+Rails.application.routes.default_url_options = {
```

Comment:
this is for grabbing images, I think I will implement a CDN to make the image retrieval much quicker